crates/ruff_server/src/fix.rs

```diff
@@ -26,33 +26,37 @@ pub(crate) fn fix_all(
     linter_settings: &LinterSettings,
     encoding: PositionEncoding,
 ) -> crate::Result<Fixes> {
-    let document_path = query.file_path();
     let source_kind = query.make_source_kind();
 
     let file_resolver_settings = query.settings().file_resolver();
+    let document_path = query.file_path();
 
     // If the document is excluded, return an empty list of fixes.
-    if let Some(exclusion) = match_any_exclusion(
-        document_path,
-        &file_resolver_settings.exclude,
-        &file_resolver_settings.extend_exclude,
-        Some(&linter_settings.exclude),
-        None,
-    ) {
-        tracing::debug!(
-            "Ignored path via `{}`: {}",
-            exclusion,
-            document_path.display()
-        );
-        return Ok(Fixes::default());
-    }
+    let package = if let Some(document_path) = document_path.as_ref() {
```

Comment:
The way I implemented it here is that an untitled file can never be excluded. However, that means that an `exclude = ["*"]` won't apply where users might expect that it should. 

But other than that, I think not handling it in exclusion makes sense because we don't know in which folder the document will be stored.

Reply:
Makes sense.